lib/ansible/modules/remote_management/redfish/redfish.py

```diff
@@ -0,0 +1,331 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2017-2018 Dell EMC Inc.
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'metadata_version': '1.1'}
+
+DOCUMENTATION = '''
+---
+module: redfish
+version_added: "2.7"
+short_description: Out-Of-Band management using Redfish APIs
+description:
+  - Builds Redfish URIs locally and sends them to remote OOB controllers to
```

Comment:
I would describe *what* this module does, rather than *how* it does it first.

> Manages out of band controllers using the redfish API

Reply:
+1, Think of the user as not being aware of how redfish works, but as someone who needs to accomplish a task.

Reply:
Makes sense, thank you both. Added to fix queue.